spec/classes/mod/ldap_spec.rb

```diff
@@ -28,11 +28,11 @@
           ldap_trusted_global_cert_file: 'ca.pem',
           ldap_trusted_global_cert_type: 'CA_DER',
           ldap_trusted_mode: 'TLS',
-          ldap_shared_cache_size: '500000',
-          ldap_cache_entries: '1024',
-          ldap_cache_ttl: '600',
-          ldap_opcache_entries: '1024',
-          ldap_opcache_ttl: '600',
+          ldap_shared_cache_size: 500000,
```

Comment:
Rubocop is unhappy about this and wants this
```suggestion
          ldap_shared_cache_size: 500_000,
```

Reply:
done